content/marketing/component-based.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Component-Based
+order: 1
```

Comment:
Used to decide the ordering of the 3-column marketing content. `order` might not be the best name for this and suggestions are welcome! I briefly considered `rank`.

Reply:
I think order works for this. Kind of like flexbox order.